ajax/libs/react-datagrid/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "react-datagrid",
+  "version": "3.0.0",
+  "filename": "index.js",
+  "description": "React DataGrid",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/zippyui/react-datagrid.git"
+  },
+  "keywords": [
+    "grid",
+    "datagrid",
+    "data-grid",
+    "widget",
+    "table",
+    "list",
+    "react-component",
+    "react"
+  ],
+  "author": {
+    "name": "ZippyUI",
+    "email": "contact@zippyui.com"
+  },
```

Comment:
It's a little bit weird that if you `npm install react-datagrid` the `package.json` which under the `react-datagrid` folder, it show the author is ZippyUI.
I think that follow the origin source from npm which release by author zippyui is better than `Radu Brehar`.